src/Views/HardwareView.vala

```diff
@@ -179,24 +190,7 @@ public class About.HardwareView : Gtk.Grid {
 
         // Graphics
         try {
```

Comment:
This is now an unreachable catch clause since the regex is in a separate method

Reply:
This can still error if there isn't a org.gnome.SessionManager instance on the bus. Is that a case we should worry about? My thinking is "probably not", but I'll leave that up to you.

Reply:
In that case you could put this inside of the try/catch for establishing the connection to the session manager, but the compiler currently complains because this try/catch won't ever catch anything